### DISCUSSION TITLE
`pac solution pack --localize` does not pack `.resx` files content into `customization.xml`

### DISCUSSION THREAD

**Discussion by szmarci** [microsoft/powerplatform-build-tools#975] [Q&A]
When packing the solution with `pac solution pack --zipfile "C:\path\solution_packed.zip " --folder "C:\path\unpacked" -p managed -loc` the Resource files not merged into the packed solution. 

However using `SolutionPackager.exe /action:pack /folder:C:\path\unpacked /zipfile:C:\path\solution_packed.zip /packagetype:managed /localize /sourceLoc` Resource files are correctly merged into the `customization.xml`.

Is this not supported anymore?

**Comment by tehcrashxor** [comment]
Just as SolutionPackager.exe requires both `/localize` and `/sourceLoc`, pac requires both `--localize` and `--sourceLoc`.

Two notes
1. It appears that both the [SolutionPackager.exe Docs](https://learn.microsoft.com/en-us/power-platform/alm/solution-packager-tool#solutionpackager-command-line-arguments) and [PAC Docs](https://learn.microsoft.com/en-us/power-platform/alm/solution-packager-tool#solutionpackager-command-line-arguments) are incorrect for the sourceLoc argument.  They both claim `Valid only on extract`, which is not correct.  
2. pac does not work by specifying _just_ `--sourceLoc` or `-src`, but requires specifying either `auto` or a locale value.
